src/style.css

```diff
@@ -1 +1,63 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-weight: bold;
```

Comment:
There is a duplicate `font-weight` property in this `@font-face` rule. Only the last one (bold) will be used, and the previous one (300) will be ignored. You should remove the first `font-weight` declaration to avoid confusion and clean up the code.